src/Pact/Repl/Lib.hs

```diff
@@ -132,6 +133,8 @@ replDefs = ("Repl",
        "Set environment gas price to PRICE."
      ,defZRNative "env-gasrate" setGasRate (funType tTyString [("rate",tTyInteger)])
        "Update gas model to charge constant RATE."
+     ,defZRNative "env-gasreal" setGasReal (funType tTyString [])
```

Comment:
Per the other Lib functions style, make this something like `env-gasmodel` where no args displays what gas model is operational, and a String argument allows selection of a gas model e.g. "cost" for cost/table model, "fixed" for fixed, and error on unrecognized etc.